src/main/scala/circt/stage/phases/CIRCT.scala

```diff
@@ -55,6 +55,14 @@ private object Helpers {
       case true  => Seq.empty
     }
   }
+
+  class LoggerShim(logger: Logger) extends firtoolresolver.Logger {
+    def error(msg: String): Unit = logger.error(msg)
+    def warn(msg:  String): Unit = logger.warn(msg)
+    def info(msg:  String): Unit = logger.info(msg)
+    def debug(msg: String): Unit = logger.debug(msg)
+    def trace(msg: String): Unit = logger.trace(msg)
+  }
 }
```

Comment:
This is already in a private object, so no need to make this private nor be aggressive with marking the methods private. Looks good.

Reply:
Indeed! Thanks for checking